packages/ui-causal-graph-editor/docs/changelog.md

```diff
@@ -2,6 +2,10 @@
 title: Changelog
 ---
 
+## Next
```

Comment:
Should be NEXT or the parser won't pick it up

Reply:
Good catch